examples/quantizing_moe/deepseek_v4_pro_example.py

```
@@ -0,0 +1,144 @@
+# Load dataset and preprocess.
+ds = load_dataset(
+    DATASET_ID,
+    split=f"{DATASET_SPLIT}[:{NUM_CALIBRATION_SAMPLES}]",
+)
+ds = ds.shuffle(seed=42)
+
+
+def preprocess(example):
+    # DeepSeek-V4 does not have a traditional chat template.
+    # Encode manually per https://huggingface.co/deepseek-ai/DeepSeek-V4-Flash/tree/main/encoding
+    BOS = "<｜begin▁of▁sentence｜>"
```

Ruff (0.15.18)

[warning] String contains ambiguous `｜` (FULLWIDTH VERTICAL LINE). Did you mean `|` (VERTICAL LINE)?

(RUF001)